js/renovation/ui/editors/internal/editor.tsx

```diff
@@ -187,9 +187,14 @@ export class Editor extends JSXComponent<EditorPropsType>() {
 
   get validationErrors(): Record<string, unknown>[] | null | undefined {
     const { validationErrors, validationError } = this.props;
-    let allValidationErrors = validationErrors;
```

Comment:
if we need just to change `validationErrors` object reference, we can only change this row to:
`let allValidationErrors = validationErrors ?? [...validationErrors];`

Reply:
right